src/openrct2/management/Award.cpp

```diff
@@ -65,7 +65,12 @@ static constexpr const rct_string_id AwardNewsStrings[] = {
     STR_NEWS_ITEM_BEST_GENTLE_RIDES,
 };
 
-Award gCurrentAwards[MAX_AWARDS];
+std::vector<Award> gCurrentAwards;
```

Comment:
```suggestion
static std::vector<Award> _currentAwards;
```
Doesn't seem to be a global anymore.